__tests__/commands/install.js

```diff
@@ -460,7 +460,8 @@ test.concurrent('install should resolve circular dependencies 2', (): Promise<vo
 });
 
 
-test.concurrent('install should respect NODE_ENV=production', (): Promise<void> => {
+// don't run this test in `concurrent`, it will affect other tests
+test('install should respect NODE_ENV=production', (): Promise<void> => {
   const env = process.env.NODE_ENV;
   process.env.NODE_ENV = 'production';
```

Comment:
It's unfortunate that we have a test that relies on global state :( Can we not somehow inject this into the code that needs it, rather than relying on global variables?